design/ordinal-crm.Rmd

```diff
@@ -0,0 +1,453 @@
+---
+title: "Design of ordinal CRM"
+author: "John Kirkpatrick and Wojciech Wójciak"
+date: "Last run on `r Sys.Date()` by `r Sys.info()[['user']]` on `r Sys.info()['nodename']`"
+output:
+  pdf_document:
+    number_sections: true
+  html_document:
+    number_sections: true
+---
+
+```{r setup, include=FALSE}
+library(ggplot2)
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# The statistical model and theory about it.
+
+The patient's toxicity response is not binary (no DLT, DLT) but categorical (for example: no DLT,
+sub-DLT, DLT). Furthermore, multicategory toxicity responses are treated as ordinal due to their
+ordinal characteristics such as a monotone trend.
+
+## The model: Cumulative Link Models
+
+These type of responses can be modeled using Generalized Linear Model (GLM), where the model is
+expressed in terms of the cumulative response probabilities. Hence, the name: Cumulative Link Model,
+or CLM. The CLM assumes the form: $$
+h[p_{ij}] = h[P(Y_i \leq j)] = \alpha_j + \mathbf x_i^T \boldsymbol \beta, \qquad j = 1, \ldots, c - 1; \, i \geq 1
+$$ where $c \in \{2, 3, \ldots \}$ denotes a number of toxicity categories on ordinal scale, and the
+components of the model are as follows:
+
+1.  $h$ - link function. The most common link functions are "logit" and "probit", which result in
+    Cumulative Logit Model and Cumulative Probit Model respectively.
+2.  $Y_i$ - random variable that represents the response outcome category for subject $i \geq 1$,
+    i.e. $supp(Y_i) = \{1, 2, \ldots, c\},\, i \geq 1$. Probabilities
+    $P(Y_i = j),\, j = 1, \ldots, c$, for $i \geq 1$, are called category probabilities, and the
+    cumulative response probabilities for subject $i \geq 1$ are defined as
+    $$p_{ij} = P(Y_i \leq j) = P(Y_i = 1) + \cdots + P(Y_i = j), \qquad j = 1, \ldots, c; ~ i \geq 1,$$
+3.  $\mathbf x_i$ - a vector of covariates corresponding to main effects and interactions for
+    subject $i \geq 1$.
+4.  $\alpha_j$, $j = 1, \ldots, c - 1$ are "cutpoints" and $\boldsymbol\beta$ is a vector of effects
+    (without intercept term). Although each response category has a corresponding cutpoint, the
+    regression coefficients $\boldsymbol\beta$ are constant across categories. Each cumulative logs
+    has its own intercept ("cutpoint"). Furthermore, if link function $h$ is strictly increasing
+    function of $P(Y_i \leq j)$, $i \geq 1$ (such as logit or probit), then \begin{equation}
+      \label{alpha_ordering0}
+      -\infty < \alpha_1 < \alpha_2 < \cdots < \alpha_{c-1} < \infty,
+    \end{equation} because $P(Y_i \leq j)$ increases in $j$ at any fixed $\mathbf x_i$.
```

Comment:
```suggestion
    function of $P(Y_i \leq j)$, $i \geq 1$ (such as logit or probit), then 
    \begin{equation}
      \label{alpha_ordering0}
      -\infty < \alpha_1 < \alpha_2 < \cdots < \alpha_{c-1} < \infty,
    \end{equation} 
    because $P(Y_i \leq j)$ increases in $j$ at any fixed $\mathbf x_i$.
```